docs/sdks/flutter/EXAMPLES.md

```diff
@@ -20,7 +20,7 @@ Account account = Account(client);
 
 Response user = await account.create(userId: '[USER_ID]', email: 'me@appwrite.io', password: 'password', name: 'My Name');
  
-Response session = await account.createSession(email: 'me@appwrite.io', password: 'password');
+Response session = await account.createEmailSession(email: 'me@appwrite.io', password: 'password'); //Create Email And Password Session
```

Comment:
To remain consistent with the previous code example and avoid a comment for self-explanatory code, let's remove the comment.